packages/cli/src/cmds/sequenceDiagramDiff.ts

```diff
@@ -50,6 +53,12 @@ export const builder = (args: yargs.Argv) => {
     describe: 'code objects to exclude from the diagram',
   });
 
+  args.option('skip-validation', {
```

Comment:
This can be `validate` and when the option is `--no-validate`, the option will be `false`. I think this is called a `flag`, when the option does not require an argument because it's either off or on. The default of course will be `true`.

Reply:
Addressed